python/ray/experimental/streaming/batched_queue.py

```diff
@@ -177,7 +170,7 @@ def _wait_for_reader(self):
     def _read_next_batch(self):
         while (self.prefetch_batch_offset <
                self.read_batch_offset + self.prefetch_depth):
-            plasma_prefetch(self._batch_id(self.prefetch_batch_offset))
+            ray.wait(ray.ObjectID(self._batch_id(self.prefetch_batch_offset)))
```

Comment:
This actually has pretty different semantics since prefetch is non-blocking.

Though maybe ray.wait(timeout=0) would achieve the same effect?

Reply:
Either way not a big deal since this example is deprecated.

Reply:
Ah interesting. Can we just call wait with timeout=0?